build.js

```diff
@@ -196,14 +197,32 @@ const rootDir = resolve(__dirname, './src');
 const buildDir = resolve(__dirname, './build');
 let modelFileIndex = [];
 
+/**
+ * Returns true if the Concerto model is compatible with the version of concerto core being used
+ * based on a comment like: // requires: concerto-core:0.82
+ * @param {string} concertoVersion the current version of concerto-core, listed in package.json
+ * @param {*} modelText the CTO model text
+ */
+function isCompatible(concertoVersion, modelText) {
+    const regex = /^\/\/.*requires:.*concerto-core:(.*)$/gm;
+    let m;
+    let versionRange = null;
+    while ((m = regex.exec(modelText)) !== null) {
+        if (m.index === regex.lastIndex) {
+            regex.lastIndex++;
+        }
+        if(m && m.length > 1) {
+            versionRange = m[1];
+        }
+    }
+    return versionRange ? semver.satisfies( concertoVersion, versionRange ) : true;
```

Comment:
To be clear, if there is no explicit version range comment, then the model is assumed to be compatible.